frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -43,57 +43,111 @@ export class ImportWizard extends Wizard {
+      await this.addVmNetworks(networkResources);
+    }
+    await this.next();
+  }
+
+  async importDiskStep(storageResources: Disk[]) {
+    // Binding storage disks
+    // First update disks that come from the source VM
+    await this.updateImportedDisks();
+    // Adding disks if any
+    if (storageResources) {
+      await this.addVmStorage(storageResources);
+    }
+    await this.next();
+    // CloudInit page is in read-only mode
+    await this.next();
+  }
+
   /**
    * Edits attributes of a NICs that are being imported from source VM.
    */
-  async updateImportedNICs() {
+
+  async updateNic(nic: Network) {
+    const nicDialog = new NetworkInterfaceDialog();
+    await clickKebabAction(nic.name, KEBAP_ACTION.Edit);
+    await waitForNoLoaders();
+    const networks = await nicDialog.getNetworks();
+    if (networks.length > 0) {
+      await nicDialog.selectNetwork(networks[networks.length - 1]);
+      const err = await checkForError(view.errorHelper);
+      if (err) {
+        return err;
+      }
+    } else {
+      throw Error('No available networks to assign imported NICs');
+    }
+    await click(view.confirmActionButton);
+    await waitForNoLoaders();
+    return null;
+  }
+
+  async getImportedNics() {
     const rows = await tableRows();
-    let importedNICs = rows.map((line) => {
+    return rows.map((line) => {
       const cols = line.split(/\t/);
       return {
         name: cols[networkTabCol.name],
       };
     });
+  }
+
+  async updateImportedNICs() {
+    let importedNICs = await this.getImportedNics();
     // TODO: This is horrible, but unfortunately no better way to dynamically extract only device names
     // without using ElementArrayFinder, which on the other hand may cause NoStaleElement Exceptions
     importedNICs = importedNICs.filter((_, i) => i % 3 === 0);
-
-    const NICDialog = new NetworkInterfaceDialog();
-    await asyncForEach(importedNICs, async (NIC) => {
-      await clickKebabAction(NIC.name, 'Edit');
-      await waitForNoLoaders();
-      const networks = await NICDialog.getNetworks();
-      if (networks.length > 0) {
-        await NICDialog.selectNetwork(networks[networks.length - 1]);
-      } else {
-        throw Error('No available networks to assign imported NICs');
-      }
-      await click(view.confirmActionButton);
-      await waitForNoLoaders();
+    await asyncForEach(importedNICs, async (nic) => {
+      await this.updateNic(nic);
     });
   }
 
   /**
    * Edits attributes of Disks that are being imported from source VM.
    */
-  async updateImportedDisks() {
+  async updateDisk(disk: Disk) {
+    const diskDialog = new DiskDialog();
+    await clickKebabAction(disk.name, KEBAP_ACTION.Edit);
+    await waitForNoLoaders();
+    // This if is required to workaround bug 1814611.
+    // NFS is not supported for conversion PV and HPP should be used instead.
+    if (disk.name === 'v2v-conversion-temp' && STORAGE_CLASS === 'nfs') {
```

Comment:
this if was removed recently

Reply:
Do you mean that NFS issue is fixed and conversion PV can use NFS backend instead of HPP?

Reply:
this was changed recently, conversion PV is not necessary  anymore https://github.com/openshift/console/pull/6574

Reply:
Removed that